digitalocean/hacktoberfest
# Prepare and update closing state

# Description

DODX 1735 - Preparing and updating the site and its closing state logic ready for November 1st.

Key changes:

Login button CTA changes to "View your profile"
Events page removes references to create an event, rsvp links
Registration is closed, replaced with a "see you next year" page

# Test process

In the event active state:

- Confirm homepage renders as expected, "Start hacking" CTA
- Confirm you can register correctly
- Confirm you can edit your profile details correctly
- Confirm events page shows create event button & rsvp links

In the event ended state:

- Confirm homepage renders as expected, "View your profile" CTA
- Confirm you cannot register as a new user
- Confirm you can view your profile if already registered
- Confirm you can edit your profile details correctly
- Confirm events page does not show create event button or rsvp links

# Requirements to merge
